static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,243 @@
 # fetch
 
-Fetches data files from the remote storage to the local cache.
+Get files that are under DVC control from remote storage into the local cache.
+Fetching means downloading from the
+[remote](https://dvc.org/doc/commands-reference/remote#description) unless the
+files in question already exist in the local cache, in which case nothing is
+done.
 
-The set of data files to fetch (usually it means downloading from the remote
-storage) is determined by analyzing all `.dvc` files in the current branch,
-unless `--all-branches` is specified.
-
-The command fetches only outputs of a specific stage if dvc file is specified
-`dvc push data.zip.dvc`.
-
-See `dvc remote`, `dvc config` and 
-[remote storages](https://dvc.org/doc/get-started/configure)
-for more information on how to configure the remote storage.
+## Synopsis
 
 ```usage
-    usage: dvc fetch [-h] [-q] [-v] [-j JOBS] [-r REMOTE] [-a]
-                     [-T] [-d]
+    usage: dvc fetch [-h] [-q | -v] [-j JOBS]
+                     [--show-checksums] [-r REMOTE]
+                     [-a] [-T] [-d] [-R]
                      [targets [targets ...]]
-
+    
+    Fetch data files from the cloud.
```

Comment:
we should make a PR to DVC and fix the command description ... not the best one :(

Reply:
(Leaving this for last but sounds easy, I can do it.)

Reply:
I've changed the DVC code already. Actually, let's just remove it from here. Considering that we have a better summary right a few lines above, this one looks redundant.

Reply:
Done.